### ISSUE TITLE
Fix "LOW" vulnerabilities found via huskyCI

### ISSUE THREAD

**Issue by rafaveira3** [globocom/prettylog#6]
Hey! I have just ran [huskyCI](https://github.com/globocom/huskyCI) and a few low vulnerabilities were found in `prettylog`. All of them are related to `Errors unhandled`:

```
[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/main.go
[HUSKYCI][!] Line: 54
[HUSKYCI][!] Code: app.Run(os.Args)

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/main.go
[HUSKYCI][!] Line: 59
[HUSKYCI][!] Code: cli.ShowAppHelp(ctx)

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 54
[HUSKYCI][!] Code: buffer.WriteString(value)

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 55
[HUSKYCI][!] Code: buffer.WriteString(SEPARATOR)

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 63
[HUSKYCI][!] Code: buffer.WriteString(color.Sprint(field[0]))

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 65
[HUSKYCI][!] Code: buffer.WriteString(field[0])

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 67
[HUSKYCI][!] Code: buffer.WriteString(FIELD_SEPARATOR)

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 68
[HUSKYCI][!] Code: buffer.WriteString(field[1])

[HUSKYCI][!] Language: Go
[HUSKYCI][!] Tool: GoSec
[HUSKYCI][!] Severity: LOW
[HUSKYCI][!] Confidence: HIGH
[HUSKYCI][!] Details: Errors unhandled.
[HUSKYCI][!] File: /go/src/code/prettifiers/default.go
[HUSKYCI][!] Line: 69
[HUSKYCI][!] Code: buffer.WriteString(SEPARATOR)

[HUSKYCI][SUMMARY] Go -> huskyci/gosec:2.0.0
[HUSKYCI][SUMMARY] High: 0
[HUSKYCI][SUMMARY] Medium: 0
[HUSKYCI][SUMMARY] Low: 9

[HUSKYCI][SUMMARY] Total
[HUSKYCI][SUMMARY] High: 0
[HUSKYCI][SUMMARY] Medium: 0
[HUSKYCI][SUMMARY] Low: 9
[HUSKYCI][SUMMARY] NoSecHusky: 0

[HUSKYCI][*] Some LOW/INFO issues were found :|
``` 

Do you think it would be a good idea to consider them? Just let me know if you would like to receive a PR fixing these issues! 🙃

**Comment by MatheusMuriel** [comment]
I think this is a good idea. Because its possible exploit buffer flaws.   
In a log context this can be dangerous.  
I can work on this issue :smile:

**Comment by mbenford** [comment]
It only took 2 years, but this issue has been finally addressed in e3a774b2dd8d8314912637c3a31b0908af683f85. :)
